devel/strip7/Portfile

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    strip7
+
+# Xcode 8.1 - matching cctools
+version                 895
+revision                7
```

Comment:
`revision` should be starting at zero for a brand new port.

Reply:
@herbygillot I can do that, but I suspect this corresponds to actually used cctools time ago. Doesn’t matter?

Reply:
If the `name` and `version` of this new port is exactly the same as a port that _used_ to exist, then yes, it would actually make sense to have an incremented `revision`.  But if the port name was `cctools` and not `strip7`, then this would be considered the first instance of the `strip7` port ever and should have a rev of zero.

Reply:
Fair enough, I will set it to zero and rebase in a few min.

Reply:
UPD. Done!